Config server base url (#168)
Thiht/smocker: README.md

- `make build-docker`, `make VERSION=xxx build-docker`: build the application as a Docker image
- `make start-docker`, `make VERSION=xxx start-docker`: run a Smocker Docker image

### Caddy

If you need to test Smocker with a base path, you can use the Caddyfile provided in the repository ([Caddy v2](https://caddyserver.com/v2)):

> - `make start-release`, `make VERSION=xxx start-release`: create a released version of Smocker and launch it with **/test/** as base path

Reviewer: ```suggestion
- `make start-release`, `make VERSION=xxx start-release`: create a released version of Smocker and launch it with `/smocker/` as base path
```

> - `make start-caddy`: start Caddy to make Smocker accessible at http://localhost:8082/test/

Reviewer: ```suggestion
- `make start-caddy`: start Caddy to make Smocker accessible at http://localhost:8082/smocker/
```

- `make build-docker`, `make VERSION=xxx build-docker`: build the application as a Docker image
- `make start-docker`, `make VERSION=xxx start-docker`: run a Smocker Docker image

> ### Test Smocker with a base path

Reviewer: Je mettrais pas ça dans le README, c'est trop spécifique comme conf. Plutôt dans la doc

> If you need to test Smocker with a base path, you can use the Caddy file provided in the repository (caddy v2):

Reviewer: ```suggestion
If you need to test Smocker with a base path, you can use the Caddyfile provided in the repository ([Caddy v2](https://caddyserver.com/v2)):
```
